homework/03LessonThree.js

```diff
@@ -89,22 +88,22 @@ var problemSevenAnswer;
 
 // Problem 8:
 // Create an empty array
-var emptyArray;
+var emptyArray = [];
 
 // Problem 9:
 // Create an array with 5 elements in it
-var lengthFiveArray;
+var lengthFiveArray = [1,2,3,4,5];
 
 // DO NOT MODIFY
-var nameArray = ['bob', 'fred', 'susan'];
+var nameArray = ['bob', 'george', 'susan'];
```

Comment:
Do not modify the variables directly.

For problems 10 & 11, use your infinite knowledge of bracket and dot notation to access and assign values inside of these arrays.